client/jetpack-cloud/sections/partner-portal/primary/licenses/index.tsx

```diff
@@ -55,6 +58,8 @@ export default function Licenses( {
 	const hasFetched = useSelector( hasFetchedLicenseCounts );
 	const allLicensesCount = counts[ 'all' ];
 	const provisioningSite = getQueryArg( window.location.href, 'provisioning' ) as string;
+	const partner = useSelector( getCurrentPartner );
+	const partnerCanIssueLicense = Boolean( partner?.can_issue_licenses );
```

Comment:
Would it make sense to make the "Issue license" button a singular shared component since we re-use this across 3 different components? It might make the tooltip implementation simpler as well since it will only have to exist in one location 😄 

If we do not wish to make it a singular "IssueLicenseButton" or something similar, would it make sense to make this a hook/utility function instead?
E.g. located in `client/jetpack-cloud/sections/utils`?

Reply:
In this case, I don't feel that it needs a singular shared component, it's just a button that we disable it according to a condition. It could be in the limit of switching to a component but it doesn't have enough weight to be worth the change, to encapsulate that 2 lines. But happy to change it if we usually create this kind of small component.

> If we do not wish to make it a singular "IssueLicenseButton" or something similar, would it make sense to make this a hook/utility function instead?
> E.g. located in client/jetpack-cloud/sections/utils?

That could be a good intermediate solution. Thank you for pointing this out.